recipes/default.rb

```diff
@@ -16,3 +16,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+repo = 'qemu-ev'
```

Comment:
It was meant to make the lines shorter but that's not really needed anymore. I'll remove and update.